src/newspack-ui/scss/elements/_typography.scss

```
@@ -16,7 +16,6 @@
 	h5,
 	h6 {
 		font-family: var(--newspack-ui-font-family);
```

```suggestion
		font-family: var(--newspack-ui-font-family);
		font-weight: var(--newspack-ui-font-weight-strong);
```

This font weight should stay -- the headings should be `600` and without it, we get a weird mix of `600` and `700`.